queries/ql/highlights.scm

```diff
@@ -0,0 +1,103 @@
+[
+  "and"
+  "any"
+  "as"
+  "asc"
+  "avg"
+  "by"
+  "class"
+  "concat"
+  "count"
+  "desc"
+  "else"
+  "exists"
+  "extends"
+  "forall"
+  "forex"
+  "from"
+  "if"
+  "implies"
+  "import"
+  "in"
+  "instanceof"
+  "max"
+  "min"
+  "module"
+  "newtype"
+  "not"
+  "or"
+  "order"
+  "rank"
+  "select"
+  "strictconcat"
+  "strictcount"
+  "strictsum"
+  "sum"
+  "then"
+  "where"
+
+  (false)
+  (predicate)
+  (result)
+  (specialId)
+  (super)
+  (this)
+  (true)
```

Comment:
Thanks for the contribution. A few notes on the different group of keywords:
true, false -> `@boolean`
this, super -> `@variable.builtin`
sum,rank,instanceof,avg... -> `@function.builtin`
import -> `@include`
if, then... -> `@conditional`
and, or,select... -> `@keyword.operator`